doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -82,6 +82,49 @@ If you want to add SMP Service to advertising data, refer to the :ref:`zephyr:sm
 
 .. fota_upgrades_over_ble_additional_information_end
 
+.. fota_upgrades_over_ble_mcuboot_direct_xip_information_start
+
+FOTA updates are also supported when MCUboot is in Direct-XIP mode.
```

Comment:
I think it would be good to name this section. My suggestions:

"FOTA over Bluetooth Low Energy with MCUboot Direct-XIP mode"
or simply
"MCUboot Direct-XIP mode"

Reply:
I think we can divide the FOTA over Bluetooth Low Energy section in the following subsections:

"Build configuration for MCUboot standard mode"

"Build configuration additions for MCUboot Direct-XIP mode"

"Testing steps"

You can create a separate commit for this